amm/contracts/stable_pool/lib.rs

```diff
@@ -394,13 +406,6 @@ pub mod stable_pool {
             Ok(())
         }
 
-        /// This method is for internal use only
```

Comment:
Why remove these comments?

Reply:
This method handles all the logic behind the swap right now. I think the inline comments are enough to understand what this method does.